.github/workflows/build_release.yaml

```diff
@@ -422,56 +152,18 @@ jobs:
           GH_TOKEN: ${{ github.token }}
         run: |
             BINARIES=($(find ./releases -type f))
-            gh release create --prerelease prerelease-${{ steps.version_or_sha.outputs.version_or_sha }}-${{ github.event.inputs.prerelease_suffix }} \
-              --target ${{ github.event.inputs.ref }} --title prerelease-${{ steps.version_or_sha.outputs.version_or_sha }}-${{ github.event.inputs.prerelease_suffix }} \
+            gh release create \
+              --prerelease prerelease-${{ steps.version_or_sha.outputs.version_or_sha }}-${{ github.event.inputs.prerelease_suffix }} \
+              --target ${{ github.event.inputs.ref }} \
+              --title prerelease-${{ steps.version_or_sha.outputs.version_or_sha }}-${{ github.event.inputs.prerelease_suffix }} \
               "${BINARIES[@]}"
 
-      # to-do rename step after depcreation - matter-labs/zksolc-bin
       - name: Publish release (this repository)
-        if: (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v'))
+        if: github.ref_type == 'tag'
         env:
           GH_TOKEN: ${{ github.token }}
         run: |
             BINARIES=($(find ./releases -type f))
             gh release create ${{ steps.version_or_sha.outputs.version_or_sha }} \
               --title ${{ steps.version_or_sha.outputs.version_or_sha }} \
               "${BINARIES[@]}"
-
-      - name: Publish release
-        if: (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v'))
-        run: |
-            chmod -R +x releases/
-            cp -R releases/* public_repo/
-            cd public_repo
-            git config --global user.email "dev-robot@matterlabs.dev"
-            git config --global user.name "zksync-admin-bot2"
-            git add .
-            git commit -m "Release: ${{ steps.version_or_sha.outputs.version_or_sha }}"
-            git push
-
-      - name: Get CHANGELOG Entry
```

Comment:
Mattermost notifications are "working" just because the Slack/Mattermost tokens are doing the same thing for now. But this has to be changed to the new way of notifications through IT ticket later. In this PR, I'm removing the old way as well.